pkgs/by-name/gi/git-credential-email/git-credential-aol/default.nix

```
@@ -0,0 +1,14 @@
+{
+  lib,
+  callPackage,
+}:
+
+let
+  script = "git-credential-aol";
+in
+callPackage ../package.nix {
+  pname = script;
+  scripts = [ script ];
```

```suggestion
callPackage ../package.nix {
  pname = "git-credential-aol";
  scripts = [ "git-credential-aol" ];
```
IMO this falls into the same category as pname overuse